card/spec/lib/card/diff_spec.rb

```diff
@@ -33,111 +33,111 @@ def p_diff
   describe 'traffic light' do
     it 'is green for addition' do
       a = "a"
-      b = "a b" 
+      b = "a b"
       db = Card::Diff::DiffBuilder.new(a,b)
       expect(db.green?).to be_truthy
       expect(db.red?).to be_falsey
     end
     it 'is red for deletion' do
       a = "a"
-      b = "" 
+      b = ""
       db = Card::Diff::DiffBuilder.new(a,b)
       expect(db.green?).to be_falsey
       expect(db.red?).to be_truthy
     end
     it 'is green and red for change' do
       a = "a"
-      b = "b" 
+      b = "b"
       db = Card::Diff::DiffBuilder.new(a,b)
       expect(db.green?).to be_truthy
       expect(db.red?).to be_truthy
     end
     it 'is off for no change' do
       a = "a"
-      b = "a" 
+      b = "a"
       db = Card::Diff::DiffBuilder.new(a,b)
       expect(db.green?).to be_falsey
       expect(db.red?).to be_falsey
     end
   end
-  
-  
+
+
   describe 'summary' do
     before(:all) do
-      @opts = {:format=>:html}
+      @opts = {format: :html}
     end
-    
+
     it 'omits unchanged text' do
       a = "<p>this was the original string</p>"
-      b = "<p>this is the new string</p>" 
+      b = "<p>this is the new string</p>"
       expect(summary a, b).to eq(
         "...#{del 'was'}#{ins 'is'}...#{del 'original'}#{ins 'new'}..."
       )
     end
-    
+
     it 'no ellipsis if changes fit exactly' do
       a = "123"
-      b = "456" 
-      expect(summary a, b, :summary=>{:length=>6}).to eq(
+      b = "456"
+      expect(summary a, b, summary: {length: 6}).to eq(
         "#{del '123'}#{ins '456'}"
       )
     end
-    
+
     it 'green ellipsis if added text does not fit' do
       a = "123"
-      b = "5678" 
-      expect(summary a, b, :summary=>{:length=>6}).to eq(
+      b = "5678"
+      expect(summary a, b, summary: {length: 6}).to eq(
         "#{del '123'}#{ins '...'}"
       )
     end
-    
+
     it 'neutral ellipsis if complete change does not fit' do
       a = "123 123"
-      b = "456 456" 
-      expect(summary a, b, :summary=>{:length=>9}).to eq(
+      b = "456 456"
+      expect(summary a, b, summary: {length: 9}).to eq(
         "#{del '123'}#{ins '456'}..."
       )
     end
-    
+
     it 'red ellipsis if deleted text partially fits' do
       a = "123456"
-      b = "567" 
-      expect(summary a, b, :summary=>{:length=>4}).to eq(
+      b = "567"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.